content/en/docs/tasks/tls/managing-tls-in-a-cluster.md

```diff
@@ -37,13 +37,25 @@ these certificates will validate against the cluster root CA.
 
 ## Trusting TLS in a Cluster
 
-Trusting the custom CA from an application running as a pod usually requires
+Trusting the [custom CA](#a-note-to-cluster-administrators) from an application running as a pod usually requires
 some extra application configuration. You will need to add the CA certificate
 bundle to the list of CA certificates that the TLS client or server trusts. For
 example, you would do this with a golang TLS config by parsing the certificate
 chain and adding the parsed certificates to the `RootCAs` field in the
 [`tls.Config`](https://godoc.org/crypto/tls#Config) struct.
 
+{{< note >}}
+Even though the custom CA certificate may be included in the filesystem by kubeadm (in the
```

Comment:
is `kubeadm` mentioned anywhere else in this topic? probably sufficient to just make the note general, regardless of what installer may have injected a CA this way